modules/dists/StencilDist.chpl

```diff
@@ -127,23 +127,30 @@ config param disableStencilLazyRAD = defaultDisableLazyRADOpt;
 //
 
 /*
-  The ``stencilDist`` distribution is a variant of the :mod:`blockDist
-  <BlockDist>` distribution that attempts to improve performance for
-  stencil computations by reducing the amount of communication
-  necessary during array accesses. From the user's perspective, it
-  behaves very similarly to ``blockDist`` in terms of reads, writes,
-  and iteration.
-
-  This distribution reduces communication by creating read-only caches for
-  elements adjacent to the block of elements owned by each locale. This
-  documentation may refer to these cached regions as 'ghost cells' or 'fluff'.
-  This approach can avoid many fine-grained GETs and PUTs when performing a
-  stencil computation near the boundary of the current locale's chunk of array
-  elements. The user must manually refresh these caches after writes by calling
-  the ``updateFluff`` method. Otherwise, reading and writing array elements
-  behaves the same as a block-distributed array.
-
-  The indices are partitioned in the same way as the :mod:`blockDist
+  The ``stencilDist`` distribution is an enhanced functionality variant of the
+  :mod:`blockDist<BlockDist>` distribution. Both refer to an array partitioned
+  into blocks, but ``stencilDist`` will internally attempt to improve
+  performance for stencil computations by reducing the amount of communication
+  necessary for array accesses across locales. A ``stencilDist`` array would
+  most commonly be used in computations based on a numerical stencil formula.
+  For example, solving banded linear equations by the Gauss-Seidel or Jacobi
+  method.
+
+  Like ``blockDist``, each of the blocks in a ``stencilDist`` array are owned
+  by a (potentially) different locale. But when communication needs to occur
+  between a given block and its set of immediately neighboring or adjacent
+  blocks, the more feature-full ``stencilDist`` most likely makes it the better
+  choice. This is because, for any given block, a ``stencilDist`` transparently
+  creates a read-only cache of selected array elements from its adjacent
+  blocks, specifically those array elements from the band of array elements
+  just outside the edge of the locally-owned block. This documentation may
+  refer to this cached band of array elements as either 'ghost cells' or more
+  strictly, as 'fluff'. This approach can  avoid many fine-grained GETs and
+  PUTs when performing a stencil computation near the boundary of the current
+  locale's chunk of array elements. The user must manually refresh these caches
+  after writes by calling the ``updateFluff`` method. Otherwise, reading and
```

Comment:
"Otherwise" here sounds a bit like "if the user has not refreshed the caches." I suggest instead "Reading and writing array elements owned by the current locale".